### PR TITLE
Add reversing lab scanner

### PR DESCRIPTION
**Pull request by tanya732**
Changes

References
https://github.com/auth0/auth0-java/pull/655/files

### CODE REVIEW

File: .github/workflows/rl-secure.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      PRODSEC_TOOLS_USER:
+        required: true
+      PRODSEC_TOOLS_TOKEN:
+        required: true
+      PRODSEC_TOOLS_ARN:
+        required: true
+
+jobs:
+  checkout-build-scan-only:
+    ### if: github.event_name == 'workflow_dispatch' || (github.event_name == 'pull_request' && github.event.pull_request.merged && startsWith(github.event.pull_request.head.ref, 'release/'))
```
Comment (kishore7snehil) [inline .github/workflows/rl-secure.yml:28]: I believe this is required here

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    secrets:
+      RLSECURE_LICENSE: ${{ secrets.RLSECURE_LICENSE }}
+      RLSECURE_SITE_KEY: ${{ secrets.RLSECURE_SITE_KEY }}
+      SIGNAL_HANDLER_TOKEN: ${{ secrets.SIGNAL_HANDLER_TOKEN }}
+      PRODSEC_TOOLS_USER: ${{ secrets.PRODSEC_TOOLS_USER }}
+      PRODSEC_TOOLS_TOKEN: ${{ secrets.PRODSEC_TOOLS_TOKEN }}
+      PRODSEC_TOOLS_ARN: ${{ secrets.PRODSEC_TOOLS_ARN }}
+  release:
+    uses: ./.github/workflows/java-release.yml
+    with:
```
Comment (kishore7snehil) [inline .github/workflows/release.yml:35]: A condition is missing saying **needs : rl-scanner** so that rl-scanner job runs post release

**Review by kishore7snehil — CHANGES_REQUESTED** [review]

File: .github/workflows/rl-secure.yml (hunk 1 above)
Comment (tanya732) [reply .github/workflows/rl-secure.yml]: Yes for testing this was commented, have removed the changes

File: .github/workflows/release.yml (hunk 2 above)
Comment (tanya732) [reply .github/workflows/release.yml:33]: Yes, I missed. Thanks for pointing

**Review by tusharpandey13 — APPROVED** [review]

**Review by kishore7snehil — APPROVED** [review]
LGTM!

**Pull request merged**